includes/states.php

```diff
@@ -0,0 +1,2126 @@
+<?php
+
```

Comment:
```suggestion
// Data pulled from https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/i18n/states.php

```
May be nice to link to where we got this data from just in case we need to pull updates later on.